modal/functions.py

```diff
@@ -882,16 +881,17 @@ async def _load(self, resolver: Resolver, existing_object_id: str):
         shared_volume_mounts = []
         # Relies on dicts being ordered (true as of Python 3.6).
         for path, shared_volume in self._shared_volumes.items():
-            # TODO: check paths client-side on Windows as well.
-            path = Path(path).as_posix()
-            abs_path = os.path.abspath(path)
+            path = PurePath(path).as_posix()
+            abs_path = posixpath.abspath(path)
```

Comment:
nit: this will still return weird stuff on windows, e.g. `C:\\foo\bar\my/posix/path`... It will technically work since all we do is check if it is absolute or match some exact paths, but I think a "cleaner" way might be to use `PurePosixPath()` and then do `.is_absolute()` for the absoluteness check (As the input paths in the shared_volume dict should all be posix paths to begin with)